portfolio/src/main/webapp/index.html

```diff
@@ -8,12 +8,33 @@
   </head>
   <body>
     <div id="content">
+
+
       <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
+      <p>This is Eddy's portfolio.</p>
+      <p>Hello! My name is Eddy Nkere and my hometown is Charlottesville Virginia.</p>
+      <p>One thing that I am quite passionate about is the story line of syfy characters in movies and TV-shows.</p>
+      <p> One of them is the Mad Titan and the Black Order</p>
+      <img src="img/thanos2.jpg" width = "500" height= "300"/>
```

Comment:
Consider using CSS for height and width. These img tag height and width attributes are honestly not used much anymore.

Typically anything layout or style related goes in CSS. You can do it in a later PR if you'd like.

Reply:
I will try using CSS next time!